storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala

```diff
@@ -86,31 +89,30 @@ class MongoStore[K: MongoValue, V: MongoValue: Manifest] (
 
   override def get(key: K): Future[Option[V]] = futurePool {
     col.findOne(MongoDBObject(keyName -> key)).flatMap { valueObject => {
-        getValue(valueObject)
+        getValue(valueObject).toOption
       }
     }
   }
 
-  protected def getValue(valueObject: MongoDBObject): Option[V] = {
+  protected def getValue(valueObject: MongoDBObject): Try[V] =
```

Comment:
Sorry I overlooked that before.
On Wed, Aug 24, 2016 at 19:51 Ben Fradet <notifications@github.com> wrote:

> In
> storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala
> <https://github.com/twitter/storehaus/pull/324#discussion_r76183627>:
>
> >        }
> >      }
> >    }
> >
> > -  protected def getValue(valueObject: MongoDBObject): Option[V] = {
> > +  protected def getValue(valueObject: MongoDBObject): Try[V] =
>
> Yeah, that's what I wanted your input on earlier.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/twitter/storehaus/pull/324/files/ec02fb3c5fc5b7878aa97b80d3f6873fedd2e263#r76183627>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEJdhQgTrOLaYVQoWj6ngIrSaNk8Ee8ks5qjS1TgaJpZM4Jefzw>
> .
>